### PR TITLE
Update README to include S2I docs

### PR DESCRIPTION
**Pull request by mfojtik**
@bparees docs for the S2I build + removal of kubernetes plugin :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,19 +109,40 @@ github:1.11.3
...
+
+The [s2i](https://github.com/openshift/source-to-image) tool allows you to do additional modifications of this Jenkins image.
+For example, you can use S2I to copy custom Jenkins Jobs definitions, additional
+plugins or replace the default `config.xml` file with your own configuration.
+
+To do that, you can either use the standalone `s2i` tool, that will produce the
+customized Docker image or you can use OpenShift Source build strategy.
+
+In order to include your modifications in Jenkins image, you need to have a Git
+repository, with following directory structure:
```
Comment (bparees) [inline README.md]: no comma

File: README.md [hunk 2]
```diff
@@ -107,19 +109,40 @@ github:1.11.3
...
+
+* `./plugins` folder that contains binary Jenkins plugins you want to copy into Jenkins 
+* `./plugins.txt` file that list the plugins you want to install (see the section above)
+* `./configuration/jobs` folder that contains the Jenkins job definitions
+* `./configuration/config.xml` file that contains your custom Jenkins configuration
```
Comment (bparees) [inline README.md]: it's probably worth noting that if they do this, then the jenkins env variable JENKINS_PASSWORDD no longer applies (right?)

**Comment by bparees** [comment]
2 minor comments and then lgtm.

File: README.md (hunk 2 above)
Comment (mfojtik) [inline README.md]: it does apply :-)

**Pull request merged**